docs/CONTRIBUTING.md

```diff
@@ -1,38 +1,136 @@
-All contributions to this project are welcome. To propose changes, we encourage contributors to:
-1. [Fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) this project on GitHub
-1. Create a new branch, and
-1. Propose their changes by opening a [new pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests). 
+If you’re reading this section, you are probably interested in contributing to the Canonical GTFS Schedule Validator. First, thank you for your interest, and welcome to the community! This document describes the different ways you can contribute to this project, and what the processes look like. 
+This project is a community effort, and anyone interested in this project can join the community, participate in decision-making and help advance this project in different ways. There are many different ways to contribute: sharing your idea for a new feature, improving the documentation, teaching others how to use it, interacting on Pull Requests, helping answer questions, etc. We welcome and value any type of contribution, and yes: you do have something to contribute to this validator! If you are not on the [MobilityData Slack](https://docs.google.com/forms/d/e/1FAIpQLSczZbZB9ql_Xl-1uBtmvYmA0fwfm1UX92SyWAdkuMEDfxac5w/viewform) already, please join the channel #gtfs-validators to introduce yourself and share what area of the project you’re interested in working on.
 
-If you're looking for somewhere to start, check out the issues labeled ["Good first issue"](https://github.com/MobilityData/gtfs-validator/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) or [Community](https://github.com/MobilityData/gtfs-validator/issues?q=is%3Aopen+is%3Aissue+label%3Acommunity).
+We value discussions, respect, and openness in our community. All experiences are welcome, regardless of the technical knowledge. We particularly encourage people from underrepresented backgrounds in Open Source to participate. 
 
-### Issue and PR templates
-We encourage contributors to format pull request titles following the Conventional Commit Specification.
+Please read our [Contributor Code of Conduct](https://github.com/MobilityData/gtfs-validator/blob/master/CODE_OF_CONDUCT.md) before contributing to this project.
 
-## Having problems?
-Have you encountered an error? A critical step in troubleshooting is being able to reproduce the problem. Instructions to publicly reproduce errors using GitHub Actions can be found in our [guide to reproduce errors.](/docs/REPRODUCE_ERRORS.md)
+## Ways to contribute
 
-## Code template
+There are many ways to contribute to this project: improving code and documentation, teaching others, answering questions on the Slack channel, etc. Good documentation is as important as quality code. The list below is an attempt to list different ways of contributing and it is not exhaustive. If you think of any other way, let us know!
```

Comment:
the Slack channel is mentioned in a few places in this doc, sometimes with a references to the "Request to join" doc and sometimes not.  It might be worthwhile adding an explicit "Slack" section header with details on how to join and the respective channels, and then linking to that header when Slack is referenced elsewhere in the doc?

Reply:
I really like this idea! I have updated the guidelines with this suggestion.